python/TestHarness/TestHarness.py

```diff
@@ -397,6 +401,20 @@ def checkExpectError(self, output, expect_error):
       return True
 
 # PBS Defs
+
+  def checkPBSEmulator(self):
+    try:
+      qstat_process = subprocess.Popen(['qstat', '--version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+      qstat_output = qstat_process.communicate()
+    except OSError:
+      # qstat binary is not available
+      return False
```

Comment:
I wonder if we shouldn't actually error out all together and tell the user that we can't find PBS. If you don't here, then they'll get an error later when the cluster launcher tries to run `qsub` or the TestHarness tries to run `qstat`. Maybe just print an error message and quit?